src/pages/kyunghyun/Login/Login.scss

```
@@ -0,0 +1,200 @@
+@mixin flex-column {
+  display: flex;
+  flex-direction: column;
+}
+@mixin flex-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+@mixin out-box {
+  background-color: #fff;
+  border: 1px solid #dbdbdb;
+  border-radius: 3px;
+}
+
+@mixin ling-font {
+  color: #385185;
+  text-align: center;
+  text-decoration: none;
+}
+
+@mixin link-text {
+  text-decoration: none;
+  text-align: center;
+}
+
+@mixin blue-text {
+  color: #385185;
+  @include link-text;
+}
+
+.KyungHyunLogin {
+  transform: translate(-50%, -50%);
+  top: 50%;
+  left: 50%;
+  position: absolute;
+  align-items: center;
+  width: 372px;
+  height: 561px;
+
+  header {
+    @include out-box;
+    @include flex-column;
+    align-items: center;
+    width: 100%;
+    padding: 10px 0;
+    margin-bottom: 10px;
+    h1 {
+      display: block;
+      width: auto;
+      margin: 22px 12px;
+      height: 51px;
+      font-family: 'Style Script', cursive;
+      text-align: center;
+      color: #000;
+      font-size: 50px;
+      font-weight: 800;
+    }
+    .main-function {
+      @include flex-column;
+      @include flex-center;
+      align-items: center;
+      width: 360px;
+      height: 263px;
+      margin-bottom: 10px;
+      .login-form {
+        @include flex-column;
+        width: 100%;
+        .id,
+        .pw {
+          margin: 0 50px 6px;
+          padding: 9px 0 7px 8px;
+          font-size: 16px;
+          background: #fafafa;
+          border: 1px solid #dbdbdb;
+          outline: none;
+          border-radius: 3px;
+          line-height: 18px;
+        }
+        input {
+          &::placeholder {
+            font-size: 12px;
+          }
+        }
+        .login {
+          margin: 6px 50px;
+          padding: 5px 9px;
+          border: 1px solid;
+          border-radius: 4px;
+          color: #fff;
+          background-color: rgba(0, 149, 246, 0.5);
+          text-align: center;
+          cursor: default;
+        }
+        .else {
+          display: flex;
+          justify-content: space-between;
+          margin: 10px 40px 18px;
+          .line {
+            position: relative;
+            flex-grow: 1;
```

```suggestion
        }
        
        .else {
          display: flex;
          justify-content: space-between;
          margin: 10px 40px 18px;
          
          .line {
            flex-grow: 1;
```
- line은 꼭 postition relative로 자리를 잡아야 하나요?
- else 말고 더 좋은 이름 있을까요?